curated_transformers/repository/hf_hub.py

```diff
@@ -11,6 +14,84 @@
 
 from ..repository.file import LocalFile, RepositoryFile
 from .repository import Repository
+from .transaction import TransactionContext
+
+
+class HfHubFile(RepositoryFile):
+    """
+    Wraps either a remote file on a Hugging Face Hub repository
+    or a local file in the Hugging Face cache.
+    """
+
+    def __init__(self, repo: "HfHubRepository", path: str):
+        """
+        Construct a Hugging Face file representation.
+
+        :param repo:
+            The parent repository.
+        :param path:
+            The path of the remote file in the repository.
+        """
+        super().__init__()
+
+        self._repo = repo
+        self._remote_path = path
+        self._cached_path = lookup_file_in_cache(repo.name, repo.revision, path)
+
+    def _validate_model_repo(self):
+        api = HfApi()
+        # This will raise RepositoryNotFoundError/RevisionNotFoundError upon failure.
+        _ = api.model_info(repo_id=self._repo.name, revision=self._repo.revision)
```

Comment:
Are you setting `_` on purpose instead of
```suggestion
        HfApi().model_info(repo_id=self._repo.name, revision=self._repo.revision)
```
?

Reply:
Yeah, it's to indicate that we are intentionally discarding the return value.